sklearn/linear_model/tests/test_ridge.py

```diff
@@ -335,6 +335,47 @@ def test_ridge_individual_penalties():
         ridge.fit(X, y)
 
 
+@pytest.mark.parametrize(
+    "params, err_type, err_msg",
+    [
+        ({"alpha": -1}, ValueError, "alpha == -1, must be >= 0.0"),
+        (
+            {"alpha": "1"},
+            TypeError,
+            "alpha must be an instance of <class 'numbers.Real'>, not <class 'str'>",
+        ),
+        ({"max_iter": 0}, ValueError, "max_iter == 0, must be >= 1."),
+        (
+            {"max_iter": "1"},
+            TypeError,
+            "max_iter must be an instance of <class 'numbers.Integral'>, not <class"
+            " 'str'>",
+        ),
+        ({"tol": -1.0}, ValueError, "tol == -1.0, must be >= 0."),
+        (
+            {"tol": 1},
+            TypeError,
+            "tol must be an instance of <class 'float'>, not <class 'int'>",
+        ),
+        (
+            {"tol": "1"},
+            TypeError,
+            "tol must be an instance of <class 'float'>, not <class 'str'>",
+        ),
+    ],
+)
+def test_Ridge_params_validation(params, err_type, err_msg):
+    """Check the parameters validation in `Ridge`."""
```

Comment:
`snake_case` must be use to name for function, hence no uppercase even if the name is constructed from the one of a class (see [this section of PEP8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)). The name of the test (`test_X_CenterStackOp`) bellow is an exception.
 
Also and IIRC, an implicit convention is that back-quotes aren't use on single-lined doc-string.
 
```suggestion
def test_ridge_params_validation(params, err_type, err_msg):
    """Check the parameters validation in Ridge."""
```